client/package.json

```diff
@@ -11,6 +11,7 @@
     "@craco/craco": "^5.7.0",
     "@material-ui/core": "^4.9.14",
     "@material-ui/icons": "^4.9.1",
+    "@reduxjs/toolkit": "^1.4.0",
```

Comment:
Looks like we're only using this package in tests. If we don't intend to use it elsewhere, can we install it as a dev dependency?

Reply:
Ooo good catch, but I should actually be USING the toolkit, I'll make a fix.